src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -57,6 +62,28 @@ public void onLootReceived(LootReceived lootReceived) {
         }
     }
 
+    public void onWidgetLoaded(WidgetLoaded event) {
+        // special case: runelite client & loot tracker do not handle unsired loot at the time of writing
+        if (event.getGroupId() == WidgetID.DIALOG_SPRITE_GROUP_ID && isEnabled()) {
```

Comment:
I agree your version is cleaner - the awkward form is a micro-optimization since it's relatively more expensive... but still cheap enough to prefer the cleaner approach